/**:
  ros__parameters:
    rolling_size: 5
    ray_dist_threshold: 0.5
    non_drivable_class_id: 0
    lidar_topic: "/livox/lidar_pc2"
    mask_topic: "seg_mask"
    camera_info_topic: "camera_info"
    out_topic: "seg_pc2"
    frequency: 10.0
